intermediate_source/ddp_tutorial.rst

```diff
@@ -2,6 +2,13 @@ Getting Started with Distributed Data Parallel
 =================================================
 **Author**: `Shen Li <https://mrshenli.github.io/>`_
 
+Prerequisites:
+
+-  `PyTorch Distributed Overview <../beginner/dist_overview.html>`__
+-  `DistributedDataParallel API documents <https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html>`__
+_  `DistributedDataParallel notes <https://pytorch.org/docs/master/notes/ddp.html>`__
```

Comment:
The last bullet looks to be an underscore instead of a hyphen: 
![image](https://user-images.githubusercontent.com/8042156/86970853-fa448780-c124-11ea-8551-d599155c4e79.png)